src/components/search-query.jsx

```diff
@@ -1,83 +1,95 @@
 import React from 'react';
 import algoliasearch from 'algoliasearch';
 import PropTypes from 'prop-types';
+import { orderBy } from 'lodash';
 
-import { RefinementList, InstantSearch, SearchBox, Hits, ClearRefinements, Pagination, PoweredBy } from 'react-instantsearch-dom';
+import {
+  RefinementList, InstantSearch, SearchBox, ClearRefinements, Pagination,
+  connectHits, PoweredBy, connectStateResults,
+} from 'react-instantsearch-dom';
 import webComponent from '../web-component';
 
 const searchClient = algoliasearch('H897LKXYG1', '50a4f124e0d934cac92e79ece376316a');
 
-const HitComponent = ({ hit }) => (
-  <div>
-    { hit.last_name ?
+const Hits = ({ hits }) => (
```

Comment:
As we discussed over video at hack night, with the growing complexity of this file, it'd be great if we could break the `Hits` component out into its own file, and import it back here into the parent. This includes all logic and data relating to `Hits`:  the `connectHits` function, the PropTypes, etc.

That way, we can keep this `SearchQuery` component more straightforward and clean, where we can easily see the structure of the children components and any logic that is applied to the rendering of those children.